pom.xml

```diff
@@ -98,7 +96,7 @@
         <alfresco.mm.version>1.2.1</alfresco.mm.version>
 
         <!-- Alfresco Desktop Sync Service Module -->
-        <alfresco.desktop-sync.version>3.0</alfresco.desktop-sync.version>
+        <alfresco.desktop-sync.version>3.1-EGATE55</alfresco.desktop-sync.version>
```

Comment:
Is this still the expected version for the sync services module?

Reply:
Yes, they are working on 3.0.1 bug-fix and they don't know when they can release 3.1.

Reply:
Ideally we'd get dsync 3.1-RC1 ... if not, please confirm that the tag created will not be deleted.